tests/functional/d/distutil_regression_import_error.py

```diff
@@ -0,0 +1,13 @@
+"""
+Regression test to check that distutils can be imported
+See https://github.com/PyCQA/pylint/issues/73
+
+See also:
+https://github.com/PyCQA/pylint/issues/2955
+https://github.com/PyCQA/astroid/pull/1321
+"""
+import distutils.version
+from distutils.util import strtobool
+
+print(distutils.version)
+print(strtobool)
```

Comment:
DIdn't we also want to have a test to see that `distutils` still emitted `no-member` for incorrect imports? We might want to add one case here.

Reply:
22524d2